automatic/paint.net/tools/ChocolateyInstall.ps1

```diff
@@ -1,23 +1,28 @@
 ﻿$ErrorActionPreference = 'Stop'
 
-$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$toolsPath = Split-Path -parent $MyInvocation.MyCommand.Definition
+$silentArgs     = "/quiet /norestart /l*v `"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
```

Comment:
Try avoiding creating more variables than necessary. Please move this into the `packageArgs` hashtable.

Reply:
Why did you resolve this comment? You have not made the change I requested.